src/resolve-template.js

```diff
@@ -0,0 +1,90 @@
+var path = require("path");
+var fileSystem = require('fs');
+var readdir = require('recursive-readdir');
+var ContextElementDependency = require('webpack/lib/dependencies/ContextElementDependency');
+var assign = Object.assign || require('object.assign');
+
+var cheerio = require('cheerio');
+
+/**
+ * Generates key-value pairs or <require from="paths"> and their corresponding files.
+ */
```

Comment:
I guess You mean `compose` element with those attributes? That is a good idea, but I'm not sure how often those attributes contain plain strings in real life as opposed to expressions that You probably can't detect at build-time.